python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -654,13 +690,36 @@ def log_psf(msg):
         # Final measurement with the CRs removed.
         self.psf_source_measurement.run(detections.sources, exposure)
 
-        # PSF is set on exposure; only return candidates for optional saving.
+        # PSF is set on exposure; candidates are returned to use for
+        # calibration flux normalization and aperture corrections.
         return detections.sources, background, psf_result.cellSet
 
+    def _normalize_calibration_flux(self, exposure, bright_sources):
+        """Normalize the calibration flux, using previously-measured
+        bright sources.
+
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            Exposure to set the ApCorrMap on.
```

Comment:
It is a bit confusing to have ApCorrMap mentioned here, even though that is what it is used for. Not sure what to say instead.